### PR TITLE
Object detection script for Jetson

### PR DESCRIPTION
**Pull request by SAlexisvf**
- Fixed input image path
- Added script argument for the output directory path
- Added line to run tensorflow CPU instead of GPU
- Added some comments

This script is tested and working correctly on the Jetson Nano

### CODE REVIEW

**Comment by SAlexisvf** [comment]
@paulvazbad @diegocardozo97 For some reason there are merge conflicts with some files on the `catkin_home` folder. I dont know what those files are.

**Comment by paulvazbad** [comment]
I dont know why you had an outdated version of some files inside the ros project, that's why there was a merge conflict. However, I have already fixed it so don't worry about it.

File: object_detection/scripts/get_objects_and_coordinates.py [hunk 1]
```diff
@@ -24,31 +25,30 @@
...
     """
     global boxes, scores, classes, num, category_index, image
 
     MODEL_NAME = 'inference_graph'
-    IMAGE_NAME = image_directory
     CWD_PATH = os.getcwd()
     PATH_TO_CKPT = os.path.join(CWD_PATH,MODEL_NAME,'frozen_inference_graph.pb')
     PATH_TO_LABELS = os.path.join(CWD_PATH,'training','labelmap.pbtxt')
-    PATH_TO_IMAGE = os.path.join(CWD_PATH,IMAGE_NAME)
+    PATH_TO_IMAGE = image_directory
```
Comment (paulvazbad) [inline object_detection/scripts/get_objects_and_coordinates.py]: So now the absolute path must be given ?

Comment (diegocardozo97) [reply object_detection/scripts/get_objects_and_coordinates.py]: Well, it is not relative only anymore. Now it can be absolute or relative path.

**Pull request merged**